CONTRIBUTING.md

```diff
@@ -155,3 +155,26 @@ There is a possibility to enable golang pprof profiler.
 
 - [prow](https://prow.apps.ovirt.org/)
 - [flakefinder](https://storage.googleapis.com/kubevirt-prow/reports/flakefinder/nmstate/kubernetes-nmstate/index.html)
+
+# FAQ
+
+## The NodeNetworkState does not show the state correctly at ubuntu 18.04 nodes.
+
+In Ubuntu 18.04 they introduced netplan for the network configuration. So to make the NetworkManager you need to
+follow this steps
+
+- edit /etc/netplan with:
```

Comment:
I think it would look prettier as paragraphs, not list with code in between:

Edit `/etc/netplan` with:

...

Apply the changes:

...